components/usage/pkg/apiv1/usage.go

```diff
@@ -8,20 +8,53 @@ import (
 	context "context"
 
 	v1 "github.com/gitpod-io/gitpod/usage-api/v1"
+	"github.com/gitpod-io/gitpod/usage/pkg/db"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/timestamppb"
+	"gorm.io/gorm"
 )
 
 var _ v1.UsageServiceServer = (*UsageService)(nil)
 
 type UsageService struct {
+	conn *gorm.DB
 	v1.UnimplementedUsageServiceServer
 }
 
 func (us *UsageService) GetBilledUsage(ctx context.Context, in *v1.GetBilledUsageRequest) (*v1.GetBilledUsageResponse, error) {
```

Comment:
I didn't spot this before but I'd call the RPC `ListUsage` because it returns a collection of records

Reply:
`ListUsage` or `LIstBilledUsage`?

Reply:
I guess `ListBilledUsage` to keep it similar. But generally, a Service should only return one type of Resource. So here, we'd normally have a BilledUsageService, which returns `BilledUsage` records but no point changing now.